### ISSUE TITLE
Accomodating non-boss special monsters in the monster utility

### ISSUE THREAD

**Issue by thomastj8991** [ninjawithkillmoon/gloomhaven-companion#32]
# Problem
Several scenarios have "boss" monsters that are just regular monsters with other attributes (increased health, shield, etc.). For example, see scenarios number 67 or 87.

Right now, there isn't a way to accommodate these types of monsters (that I know of). I propose the following solution below.

# Solutions
When you click on the monster level button today, it cycles from 0 through 7. There could be one more option after 7 called "Other" or "Special" that then allows you to customize the amount of health assigned to that monster. Maybe you would see a prompt or you could type into the HP field to change it to a specific value.

In an ideal world the scenario data would also provide this info, perhaps as a separate monster type, but this functionality should exist for things other than just the built-in scenarios so if you play custom ones or future released ones it could support that, too.

For now, we'll likely track the boss monsters separately from the app, but maybe a bit more flexibility could be added to work with these situations.

**Comment by ASunday47** [comment]
Good idea. This is also true in the second kickstarter scenario (+1 shield, health formula based on # of players).

**Comment by blemaire** [comment]
There is also a scenario where some monsters are 2 lvl higher than the others but this is not currently accounted for / possible to do. Thanks

**Comment by ninjawithkillmoon** [comment]
@blemaire When you are adding monsters (not on the All Active tab), there is a white button that shows the monster level. This defaults to the number selected at the top of the page, but if you press this button it will change the monster level for this monster only, and the health will change accordingly.

Good suggestions otherwise. We should be able to add the ability to create an enemy with a custom health value.

**Comment by blemaire** [comment]
Thanks @ninjawithkillmoon, could this be made a little clearer as it's not obvious at all?
Thanks a lot

**Comment by ninjawithkillmoon** [comment]
It's in the instructions at the bottom of the page, but I guess something could be done to make the button look more... clickable?

**Comment by blemaire** [comment]
Maybe indicating the level as `Lvl x` in the button instead of simply `x` could do the trick?
![image](https://user-images.githubusercontent.com/7140322/28308408-078c09d4-6b9e-11e7-83dc-671851ca4834.png)

**Comment by thomastj8991** [comment]
I really like @blemaire 's suggestion for adding the Lvl wording.

**Comment by blemaire** [comment]
Some scenarios also have altars or door that have hitpoints, there is currently no way to keep track of their health, could this be added?
